.env.example

```diff
@@ -1,7 +1,15 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-PORT=
-
-DB_TEST_NAME=barefoot_test
+DB_USERNAME=postgres
+DB_PASSWORD=db_password
+DB_NAME=barefoot_nomad
+DB_HOST=127.0.0.1
+PORT=""
+DB_TEST_NAME=barefoot_test
+TOKENEXPIRY=18000
+CLIENT_URL="https://firestar_backend.herokuapp.com"
+HOST="localhost"
+DATABASE_URL_TEST=""
+NODE_ENV="test"
+YOUR_EMAIL=""
+SOME_PASSWORD=""
+API_VERSION="v1"

```

Comment:
Why is `API_VERSION` an env?
please remove it and reach out to @zanio.

Reply:
This is only for development - for sending emails with localhost address to user email. In production, the full base path is the heroku url. Is this OK?

Reply:
![image](https://user-images.githubusercontent.com/28393660/63879575-e89e3980-c9c3-11e9-8117-75529ec5c648.png)

Reply:
![image](https://user-images.githubusercontent.com/28393660/63879671-16837e00-c9c4-11e9-8a40-e061b9eb4a72.png)